# def suma (numero1 , numero2):
    #suma2 = numero1 + numero2
    #return suma2
#numero1 = input ('Ingrese el primer numero')
#numero1 = int(numero1)
#numero2 = input ('Ingrese el segundo numero')
#numero2 = int(numero2)
#sumatoria = suma (numero1 + numero2) el error estaba en que los sumaba ...
#print('El resultado es', sumatoria)

def suma (a, b):
    resultado = a + b
    return resultado

num1 = input ('Ingrese un numero')
num1 = int (num1)
num2 = input ('Ingrese otro numero')
num2 = int (num2)
sumatoria = suma(num1, num2)
print (sumatoria)